<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>proceedTab</class>
 <widget class="QDialog" name="proceedTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Installation</string>
  </property>
  <widget class="QLabel" name="proceedInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-10</y>
     <width>451</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Do You Want To Proceed To Setup -|-|-|- Development Environment?</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnProceed">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Proceed</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>proceedTab</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
